lib/instrument/transformer/nskeyed.js

```diff
@@ -322,20 +409,14 @@ class Unarchive {
 class Archive {
   constructor (inputObject) {
```

Comment:
these methods cry for some docstrings

Reply:
Yes, these methods drain me out to understand them. I'll try later